app/controllers/application_controller/performance.rb

```diff
@@ -634,7 +634,7 @@ def perf_gen_data_before_wait
                              from_dt,
                              to_dt,
                              interval_type]
-      elsif %w(MiddlewareServer MiddlewareDatasource MiddlewareMessaging).include?(@perf_record.class.name.demodulize)
+      elsif %w(MiddlewareServer MiddlewareServerEap MiddlewareDatasource MiddlewareMessaging).include?(@perf_record.class.name.demodulize)
```

Comment:
what about MiddlewareServerWildfly? it sounds like it would have the same problem, no?

Reply:
Yes We'll have the same problem. Added MiddlewareServerWildfly.